modules/redpanda/options.go

```diff
@@ -47,6 +47,9 @@ type options struct {
 	// ExtraBootstrapConfig is a map of configs to be interpolated into the
 	// container's bootstrap.yml
 	ExtraBootstrapConfig map[string]any
+
+	// EnableAdminAPIAuthentication enables Admin API authentication
+	EnableAdminAPIAuthentication bool
```

Comment:
suggestion: as per comment above about exporting, should be keep this private too?